build/common.ps1

```diff
@@ -492,10 +573,23 @@ Function Invoke-ILMerge {
         [string]$Configuration = $DefaultConfiguration
     )
     $nugetArtifactsFolder = Join-Path $Artifacts "$Configuration\NuGet.Clients\NuGet.CommandLine"
-    pushd $nugetArtifactsFolder
+    $dlls = Get-ChildItem $nugetArtifactsFolder -Filter 'nuget.*.dll' | %{ $_.Name }
```

Comment:
@yishaigalatzer The intention was to have a one-liner with no need to update it every time dll structure changes.
But I understand the idea of controlling dependencies. Will revisit the method as you suggested.